lib/book.rb

```diff
@@ -0,0 +1,19 @@
+class Book
+  attr_reader :title
+
+  def initialize(info)
+    @title = info[:title]
+    @author_first_name = info[:author_first_name]
+    @author_last_name = info[:author_last_name]
+    @publication_date = info[:publication_date]
+  end
+
+  def author
```

Comment:
Similar to the Author class, you could create `@author` and `@publication_year` in the initialize (you then wouldn't have the ivars for `@author_first_name`, `@author_last_name`, and `@publication_date`)